apps/web/src/lib/reference/cost-reference-config.ts

```diff
@@ -0,0 +1,27 @@
+export const COST_REFERENCE_TYPE_INFO = {
+    cost_product: {
+        type: 'cost_product',
+        key: 'product',
+        name: 'Product (Cost)',
+    },
+    cost_category: {
+        type: 'cost_category',
+        key: 'category',
+        name: 'Category',
+    },
+    cost_resource_group: {
+        type: 'cost_resource_group',
+        key: 'resource_group',
+        name: 'Resource Group',
+    },
+    cost_type: {
+        type: 'cost_type',
+        key: 'usage_type',
+        name: 'Type',
+    },
```

Comment:
question: Oh, I thought we have only two cost variables. Are they added?

Reply:
These were added here because they are only used in the **cost widget/dashboard**. I will double-check them!